### PR TITLE
updated magic link url to remove account id as this comes from the jwt

### PR DESCRIPTION
**Pull request by srh-sloan**
Part of work on FS-1163 to implement custom domain

### CODE REVIEW

File: manifest-dev.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ applications:
     ACCOUNT_STORE_API_HOST: https://funding-service-design-account-store-dev.london.cloudapps.digital
     APPLICATION_STORE_API_HOST: https://funding-service-design-application-store-dev.london.cloudapps.digital
     NOTIFICATION_SERVICE_HOST: http://funding-service-design-notification-dev.apps.internal:8080
-    MAGIC_LINK_REDIRECT_URL: https://funding-service-design-frontend-dev.london.cloudapps.digital/account/{account_id}
+    MAGIC_LINK_REDIRECT_URL: https://frontend.dev.fundingservice.co.uk/account
```
Comment (gidsg) [inline manifest-dev.yml:13]:
```suggestion
    APPLICANT_FRONTEND_HOST: https://frontend.dev.fundingservice.co.uk
```
I think this is a bit simpler and means we can rely on the config for the path, this is what we do in the docker runner: https://github.com/communitiesuk/funding-service-design-docker-runner/blob/00381119ef3b76d75812129435c28f9396c5ae98/docker-compose.yml#L43

File: models/magic_link.py [hunk 2]
```diff
@@ -165,7 +165,7 @@ def _create_redirect_url(account: Account):
         :param account: Account instance of the user
```
Comment (gidsg) [inline models/magic_link.py:165]: can we remove this account argument now, I don't think we're using it anymore?

File: .gitignore [hunk 3]
```diff
@@ -143,3 +143,5 @@ flask_session
 .DS_Store
 
 html
+manifest-dev.yml
```
Comment (gidsg) [inline .gitignore:146]: Don't think this should be gitignore'd?

File: manifest-dev.yml (hunk 1 above)
Comment (srh-sloan) [reply manifest-dev.yml:13]: Agreed - have simplified the code around this too

File: .gitignore (hunk 3 above)
Comment (srh-sloan) [reply .gitignore:146]: Oops!

**Review by gidsg — APPROVED** [review]
LGTM 👍  🛺

**Review by adamdavies1 — APPROVED** [review]
LGTM

**Pull request merged**